Unix/doc/setup-kerberos-omi.md

```diff
@@ -0,0 +1,627 @@
+
+   - Active Directory uses Kerberos and the Lightweight Directory Access Protocol (LDAP) protocols to authenticate 
+     and authorise machines in an active directory *domain*. Though the three functions can be distributed to other servers, 
+     normally a central server, the Active Directory Controller (ADC), provides DNS, LDAP and Kerberos services for all 
+     the machines joined to the domain.  User, group and host information is provided by querying the ldap database 
+     across the entire domain. Member user login and privelege information can be set up on the domain controller, 
+     which is a single location for all the information. 
+
+     An active domain *domain* is by convention all upper case ( for example, CONTOSO.COM ). 
+
+So a given domain joined host will receive name services from myserver.contoso.com, kerberos authentication services from
+myserver.contoso.com@CONTOSO.com and user and group information from ldap/myserver.contoso.com@CONTOSO.COM, and you can 
+login with ssh -K to host/myserver.contols.com@CONTOSO.com.
+
+On the active domain controller, You will need to verify:
+
+  - Computer Account
+  - User Account
+  - Service Principals for all expected services must be added to the computer account. 
+
+  - Presence of _kerberos.mydomain.com TXT record containing the name of the kerberos realm ("CONTOSO.com"). This 
+    is used by kerberos to correcly generate the names of well known service principals, and is used heavily by both
+    MIT and heimdal kerberos implementations. This appears not to be set by default so must be set manually. *Note, not
+    setting this will result in some very mysterious problems*. 
+
+  - Reverse DNS records for registered hosts is a common problem, as those are often managed by enterprise IT 
+    or IP organizations. It is used heavily by default, but flags and configuration options are available to compensate
+    if there are problems with it. You can identify reverse dns issues by using dig to
+    check. 
+
+  - Authoratative user account with permission to join machines to domain
+
+### Identity Mapping
+
+Linux, MacOS and Unix identify users via an integer user ID (uid). That user is then given privileges according to is membership 
+in various groups, which are identified by an integer group id.  The utility id will show both of these for a given user. 
+
+In Windows the user id is in the form of an SID, which is much longer, variable in length, and heirarchical. Groups also have SIDs.
+The user information provided by the Active Directory Controller is in the form of SIDs and must be mapped to the simpler 
+Linux/MacOS/Unix format. This is done in a very simple form by winbind or sssd on the client, but there are additional packages
+such as IdM or FreeIPA that perform this operation with considerably more sophistication.
+
+We have not found it necessary to use these.
+
+## Setting up the client machine
```

Comment:
not required. The KDC/ADC is a trusted third party. That said, setting up an ADC is a whole separate topic.